src/serial.c

```diff
@@ -445,8 +445,15 @@ int serial_recieve(int (*serial_event)(char *, int, double, void *), void *data)
     struct timeval tv;
     size_t len;
 
-    tv.tv_sec = 0;
-    tv.tv_usec = 200000;
+    // [s] -> [us]
+    long long int timeout_us = p(YP_PARAM_DEVICE_TIMEOUT, 0) * 1000000;
+    if (timeout_us == 0)
+    {
+      // Default timeout before loading parameter file.
+      timeout_us = 200000;
```

Comment:
Why is the default value different from the parameter's default value.

Reply:
Just to keep the PC's read timeout value same as before this PR.